src/AuthReg.cc

```diff
@@ -21,6 +21,7 @@
 #include "auth/negotiate/Scheme.h"
 #endif
 
+#include "debug/Messages.h"
```

Comment:
With the shift to level 2 initialization messages, do we need to add this still?